<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reportabug</class>
 <widget class="QMainWindow" name="Reportabug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report a bug</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_null"/>
      <widget class="QWidget" name="page_github">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="layout_auth">
          <item>
           <widget class="QLabel" name="label_username">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_username">
            <property name="toolTip">
             <string>Your GitHub account</string>
            </property>
            <property name="placeholderText">
             <string>username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_password">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_password">
            <property name="toolTip">
             <string>GitHub account password</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_title">
          <item>
           <widget class="QLabel" name="label_title">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_title">
            <property name="toolTip">
             <string>Title of the report</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="toolTip">
           <string>Text of the report</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_gitreport">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
